tools/packaging/kata-deploy/local-build/kata-deploy-binaries.sh

```diff
@@ -112,13 +113,26 @@ install_cc_clh() {
 
 #Install cc capable guest image
 install_cc_image() {
+	export AA_KBC="${1}"
```

Comment:
Suggestion. Fallback to defaults if $1 and/ $2 aren't passed, this way you don't need to create the `install_cc_generic_image`.

```
export AA_KBC="${1:-offline_fs_kbc}"
image_type="${2:-image}"
```